components/raftstore-v2/src/fsm/peer.rs

```diff
@@ -212,8 +212,7 @@ impl<'a, EK: KvEngine, ER: RaftEngine, T: Transport> PeerFsmDelegate<'a, EK, ER,
                 }
                 PeerMsg::RaftCommand(cmd) => {
                     self.on_receive_command(cmd.send_time);
-                    // self.on_command(cmd.cmd.request, cmd.ch)
-                    unimplemented!()
+                    self.on_command(cmd.request, cmd.ch)
```

Comment:
how about add duration metrics for the `on_msgs`, we are lack of finer granularity within append log duration

Reply:
We will have a final PR to check missing metrics when everything is about finished.